server/data/data-library/src/test/java/org/apache/james/domainlist/lib/AbstractDomainListTest.java

```diff
@@ -110,7 +114,7 @@ public void deletingADomainShouldNotDeleteOtherDomains() throws DomainListExcept
         try {
             domainList.removeDomain(DOMAIN_2);
         } catch (DomainListException e) {
-
+            LOGGER.info("Ignored error", e);
```

Comment:
I can not as the exception thrown would prevent latest test instructions to be executed.